File: PlacementCellBackend/Services/Feedback/Interfaces/IEmployeeOnStudentService.cs
using PlacementCellBackend.DTOs.EmployeeFeedbackonStudent;

namespace PlacementCellBackend.Services.Feedback.Interfaces
{
    public interface IEmployeeOnStudentService
    {
        Task<IEnumerable<EmployeeFeedbackonStudentDtos>> GetAllEmployeeOnStudentsAsync();
        Task<EmployeeFeedbackonStudentDtos?> GetEmployeeOnStudentByIdAsync(int id);
        Task<bool> CreateEmployeeOnStudentAsync(EmployeeFeedbackonStudentCreateDtos employeeOnStudent);
        Task<bool> UpdateEmployeeOnStudentAsync(int id, EmployeeFeedbackonStudentCreateDtos employeeOnStudent);
        Task<bool> DeleteEmployeeOnStudentAsync(int id);
    }
}

File: PlacementicsBackend/src/main/java/com/example/placementicsbackend/services/Feedback/Interfaces/IEmployeeOnStudentService.java
package com.example.placementicsbackend.services.Feedback.Interfaces;

import com.example.placementicsbackend.dtos.EmployeeFeedbackonStudent.EmployeeFeedbackOnStudentCreateDto;
import com.example.placementicsbackend.dtos.EmployeeFeedbackonStudent.EmployeeFeedbackOnStudentDto;
import java.util.List;

public interface IEmployeeOnStudentService {

    List<EmployeeFeedbackOnStudentDto> getAllEmployeeOnStudents();

    EmployeeFeedbackOnStudentDto getEmployeeOnStudentById(int id);

    boolean createEmployeeOnStudent(EmployeeFeedbackOnStudentCreateDto employeeOnStudent);

    boolean updateEmployeeOnStudent(int id, EmployeeFeedbackOnStudentCreateDto employeeOnStudent);

    boolean deleteEmployeeOnStudent(int id);
}
